var/spack/repos/builtin/packages/python/package.py

```diff
@@ -67,6 +67,10 @@ class Python(AutotoolsPackage):
 
     extendable = True
 
+    # Variants to control dependency variants to avoid cyclical dependencies
+    variant('libxml2', default=False,
+            description='Use a gettext library build with libxml2')
```

Comment:
@becker33 

"build" -> "built"

Also, if we go this route, then I think the associated comment needs more detail (and/or a reference to an issue/PR thread which explains it in more detail).

Reply:
does Python even need libxml support in gettext or can we just decide to generally take the variant `gettext ~libxml` as dependency?